spec/expressions/comments.hrx

```diff
@@ -0,0 +1,156 @@
+<===> loud/sass/inline-closed/input.sass
+/* a */
+
+<===> loud/sass/inline-closed/output.css
+/* a */
+
+<===>
+================================================================================
+<===> loud/sass/inline-open/input.sass
+/* a
+
+<===> loud/sass/inline-open/output.css
+/* a */
+
+<===>
+================================================================================
+<===> loud/sass/indented-closed/input.sass
+/* 
+  a */
+
+<===> loud/sass/indented-closed/output.css
+/* a */
+
+<===>
+================================================================================
+<===> loud/sass/indented-closed-after/input.sass
+/* 
+  a 
+  */
+
+<===> loud/sass/indented-closed-after/output.css
+/* a 
+ * */
+
+<===>
+================================================================================
+<===> loud/sass/indented-closed-no-indent/input.sass
+/* 
+  a 
+*/
+
+<===> loud/sass/indented-closed-no-indent/error
+WARNING: This selector doesn't have any properties and won't be rendered.
+
+  ,
+3 | */
+  | ^^
+  '
+    input.sass 3:1  root stylesheet
+
+Error: expected selector.
+  ,
+3 | */
+  |  ^
+  '
+  input.sass 3:2  root stylesheet
+
+<===>
+================================================================================
+<===> loud/sass/indented-interpolation/input.sass
+/* #{a 
+  + b} */
+
+<===> loud/sass/indented-interpolation/output.css
+/* ab */
+
+<===>
+================================================================================
+<===> loud/sass/indented-open/input.sass
+/* 
+  a
+
+<===> loud/sass/indented-open/output.css
+/* a */
+
+<===>
+================================================================================
+<===> silent/sass/inline/input.sass
+// a
+
+<===> silent/sass/inline/output.css
+
+<===>
+================================================================================
+<===> silent/sass/indented/input.sass
+// 
+  a 
+
+<===> silent/sass/indented/output.css
+
+<===>
+================================================================================
+<===> silent/sass/indented-interpolation/input.sass
+// #{a 
+  + b} 
+
+<===> silent/sass/indented-interpolation/output.css
+
+<===>
+================================================================================
+<===> whitespace-comment/sass/inline/input.sass
```

Comment:
I think "comment" here is redundant. If you're trying to emphasize that the comment is being parsed as a whitespace, I'd call this `as_whitespace`.